crowbar_framework/config/application.rb

```diff
@@ -65,7 +65,5 @@ class Application < Rails::Application
     end
     # normal options
     config.crowbar = config_for(:crowbar)
-    # experimental options
```

Comment:
is this the last experimental option we have?

Reply:
yup, all others have been moving out of experimental and are now normal features :)